src/hooks/__data__/mock-data.ts

```diff
@@ -514,3 +515,166 @@ export const mockLimitRange: LimitRange = {
     ],
   },
 };
+
+export const mockedServiceAccount: ServiceAccountKind = {
+  kind: 'ServiceAccount',
+  apiVersion: 'v1',
```

Comment:
let us just enjoy the required filed here?

Reply:
I'm going to remove the non-required data :)